src/main/java/com/demo/MybatisApplication/dto/SubjectEntityDisplayDto.java

```diff
@@ -6,5 +6,5 @@
 public class SubjectEntityDisplayDto {
     private Long id;
     private String name;
-    private TeacherSubjectDisplayDto teacher;
+    private Long teacherId;
```

Comment:
We should provide the information of entire teacher while displaying Subject.

Reply:
Done